readthedocs/builds/forms.py

```diff
@@ -31,8 +31,8 @@ class Meta:
         model = Version
         fields = ['active', 'privacy_level', 'tags']
 
-    def save(self, *args, **kwargs):
-        obj = super(VersionForm, self).save(*args, **kwargs)
+    def save(self, commit=True):
+        obj = super(VersionForm, self).save(commit=commit)
         if obj.active and not obj.built and not obj.uploaded:
             trigger_build(project=obj.project, version=obj)
         return obj
```

Comment:
In this case, we should have never pushed `*args, **kwargs`, as it was just lazy. `commit=True` is more explicit and is now identical to the underlying method signature.

In cases where there are many arguments, I don't know what's best there. `*args, **kwargs` is flexible for future compatibility, so probably in that case it's fine to ignore the linting error and use the `*args, **kwargs` catch all.